content/en/docs/how-to/registry-authentication.md

```diff
@@ -7,74 +7,54 @@ weight: 1
-
-In development and testing environments, you can use [environment variables](#configure-environment-variables-to-authenticate-to-an-oci-compliant-registry) to authenticate to an OCI-compliant registry.
-{{% /alert %}}
+## Configure a credentials store
 
-## Configure Docker Credential Store for Linux
+A credentials store is a secure container that stores your credentials, such as usernames and passwords. Operating systems often include a default credentials store, such as *osxkeychain* on macOS or *wincred* on Windows. You can also use a third-party credentials store, such as [pass](https://www.passwordstore.org/).
 
-As a security best practice, you should use a credential helper with a system keychain when using local credentials to access remote repositories.
+By default, `notation` uses a credentials store during the following operations:
+- Store credentials when you log in to a registry
+- Read credentials when accessing a registry to sign, verify, or display artifacts
+- Delete credentials when you log out of a registry
+ 
+You have three options for configuring a credentials store:
+1. Use the [`notation` config.json file]({{< ref "/docs/concepts/directory-structure" >}}).
+1. Use the Docker configuration file, which is located at `~/.docker/config.json`.
+1. Use the default credentials store for your operating system.
 
-`notation` uses [Docker Credential Helpers](https://github.com/docker/docker-credential-helpers) and its [protocol](https://docs.docker.com/engine/reference/commandline/login/#credential-helper-protocol) as the credential helpers. Currently, using Docker Credential Helpers requires manual installation and configuration in Linux by following the steps below. This configuration will be simplified in Notation v1.0.0.
+`notation` will resolve the credentials store in the order shown above. If no credential store is available and no other method is used for credentials, such as environment variables, `notation login` will not store the credentials and the command will fail.
 
-Install [Docker credential helper](https://github.com/docker/docker-credential-helpers) pass.
-
-```bash
-mkdir -p ~/bin
-curl -Lo ~/bin/docker-credential-pass "https://github.com/docker/docker-credential-helpers/releases/download/v0.7.0/docker-credential-pass-v0.7.0.linux-amd64"
-chmod +x ~/bin/docker-credential-pass
-```
-
-Generate and configure GPG key for encryption. 
+{{% alert title="Important" color="important" %}}
+If you have Docker installed on your system and used `docker login` to authenticate to your OCI-compliant registry, your credentials are already stored and available to `notation`. In this case, you don't need to run `notation login` again to authenticate to your OCI-compliant registry.
```

Comment:
Hmmm the above behavior may not work if Docker is using non OS default credential store because of https://github.com/notaryproject/notation/issues/696 present in RC-7. It will be fixed in the next release 1.0.0, so wondering if we want to put a small callout here.

Reply:
I'm happy to leave this as is and wait until 1.0.0 before merging

Reply:
Agree with @zr-msft that we can defer the merging until v1 is released.

Reply:
@iamsamirzon would you mind reviewing it again, since you request change, which block the merging. Thanks.